pages/embed/card/_iscnId.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div v-if="record">
+    <ClientOnly>
+      <IscnCard v-bind="cardProps" />
+    </ClientOnly>
+    <pre
+      v-if="isShowAdjustableProps"
+      class="p-[16px] text-[10px] font-mono"
+    >{{ JSON.stringify(adjustableProps, null, ' ') }}</pre>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component } from 'vue-property-decorator'
+import { namespace } from 'vuex-class'
+
+import { isCosmosTransactionHash } from '~/utils/cosmos'
+import { ISCNRecordWithID } from '~/utils/cosmos/iscn/iscn.type'
+import { ISCN_PREFIX } from '~/constant'
+
+const iscnModule = namespace('iscn')
+
+@Component({ layout: 'blank' })
+export default class CardIscnIdPage extends Vue {
+  iscnId = ''
+
+  @iscnModule.Getter getISCNById!: (arg0: string) => ISCNRecordWithID
+
+  @iscnModule.Action fetchISCNById!: (arg0: string) => Promise<{
+    records: ISCNRecordWithID[]
+    owner: string
+    latestVersion: Long.Long
+  } | null>
+
+  @iscnModule.Action fetchISCNByTx!: (
+    arg0: string
+  ) => Promise<{ records: ISCNRecordWithID[] }>
+
+  get record() {
+    return this.getISCNById(this.iscnId)
+  }
+
+  get isShowAdjustableProps() {
+    const { dev = '0' } = this.$route.query
+    return dev === '1'
+  }
```

Comment:
Should this be behind `NODE_ENV !== 'production'` ?

Reply:
This acts as an inspector, should come in handy even in production